sink-connector/src/test/kotlin/org/neo4j/connectors/kafka/sink/Neo4jSinkRawJSONIT.kt

```diff
@@ -0,0 +1,168 @@
+                  TOPIC,
+                  "WITH __value AS person MERGE (p:Person {name: person.name, surname: person.surname})",
+              ),
+          ],
+  )
+  @Test
+  fun `should support json map`(
+      @TopicProducer(TOPIC) producer: ConvertingKafkaProducer,
+      session: Session,
+  ) {
+    producer.publish(
+        value = mapOf("name" to "Jane", "surname" to "Doe"),
+        valueSchema = SchemaBuilder.map(Schema.STRING_SCHEMA, Schema.STRING_SCHEMA).build(),
+    )
+
+    await().atMost(30.seconds.toJavaDuration()).until {
+      session
+          .run(
+              "MATCH (p:Person {name: \$name, surname: \$surname}) RETURN count(p) = 1 AS result",
+              mapOf("name" to "Jane", "surname" to "Doe"),
+          )
+          .single()["result"]
+          .asBoolean()
+    }
+  }
+
+  @Neo4jSink(
+      cypher =
+          [
+              CypherStrategy(
+                  TOPIC,
+                  "WITH __value AS persons UNWIND persons AS person MERGE (p:Person {name: person.name, surname: person.surname})",
+              ),
+          ],
+  )
+  @Test
+  fun `should support json list`(
+      @TopicProducer(TOPIC) producer: ConvertingKafkaProducer,
+      session: Session,
+  ) {
+    producer.publish(
+        value =
+            listOf(
+                mapOf("name" to "Jane", "surname" to "Doe"),
+                mapOf("name" to "John", "surname" to "Doe"),
+            ),
+        valueSchema =
+            SchemaBuilder.array(
+                    SchemaBuilder.map(
+                            Schema.STRING_SCHEMA,
+                            Schema.STRING_SCHEMA,
+                        )
+                        .build(),
+                )
+                .build(),
+    )
+
+    await().atMost(30.seconds.toJavaDuration()).untilAsserted {
+      session
+          .run(
+              "MATCH (p:Person) RETURN count(p) as result",
+          )
+          .single()["result"]
+          .asLong() shouldBe 2L
+    }
+  }
+
+  @Neo4jSink(
+      cypher =
+          [
+              CypherStrategy(
+                  TOPIC,
+                  "WITH __value AS name MERGE (p:Person {name: name})",
+              ),
+          ],
+  )
+  @Test
+  fun `should support raw string value`(
+      @TopicProducer(TOPIC) producer: ConvertingKafkaProducer,
+      session: Session,
+  ) {
+    producer.publish(
+        value = "John",
+        valueSchema = Schema.STRING_SCHEMA,
+    )
+
+    await().atMost(30.seconds.toJavaDuration()).untilAsserted {
+      session
+          .run(
+              "MATCH (p:Person) RETURN count(p) as result",
+          )
+          .single()["result"]
+          .asLong() shouldBe 1L
+    }
+  }
+
+  @Neo4jSink(
+      cypher =
+          [
+              CypherStrategy(
+                  TOPIC,
+                  "WITH __value AS age MERGE (p:Person {age: age})",
+              ),
+          ],
+  )
+  @Test
+  fun `should support raw numeric value`(
```

Comment:
maybe boolean too?
not sure about `null`

Reply:
added boolean, but skipped null.